auto_install/install.sh

```diff
@@ -25,18 +25,22 @@ piholeSetupVars="/etc/pihole/setupVars.conf"
 dnsmasqConfig="/etc/dnsmasq.d/02-pivpn.conf"
 
 dhcpcdFile="/etc/dhcpcd.conf"
-debianOvpnUserGroup="openvpn:openvpn"
+ovpnUserGroup="openvpn:openvpn"
 
 ######## PKG Vars ########
 PKG_MANAGER="apt-get"
 ### FIXME: quoting UPDATE_PKG_CACHE and PKG_INSTALL hangs the script, shellcheck SC2086
 UPDATE_PKG_CACHE="${PKG_MANAGER} update -y"
 PKG_INSTALL="${PKG_MANAGER} --yes --no-install-recommends install"
 PKG_COUNT="${PKG_MANAGER} -s -o Debug::NoLocking=true upgrade | grep -c ^Inst || true"
+CHECK_PKG_INSTALLED='dpkg-query -s'
 
 # Dependencies that are required by the script, regardless of the VPN protocol chosen
 BASE_DEPS=(git tar curl grep dnsutils grepcidr whiptail net-tools bsdmainutils bash-completion)
 
+BASE_DEPS_ALPINE=(git grep bind-tools newt net-tools bash-completion coreutils openssl)
+BASE_DEPS_ALPINE+=(util-linux openrc iptables ip6tables)
```

Comment:
why declaring same variable twice and adding values to it instead of doing it all in the same one?

Reply:
I did it to have good readability
By completely declaring the array on a single line, it would have been less readable

Reply:
maybe there's better option, but its fine by me.